### DISCUSSION TITLE
Android: Cannot select Standard Notes as app to share with

### DISCUSSION THREAD

**Discussion by rdiaz02** [standardnotes/forum#2814] [Suggestions]
In Android (version 12, on a Samsung tablet) I cannot share selected text with Standard Notes as the destination. 

This happens to me at least when I select text in:
- Web browsers:
   - Chrome
   - Firefox
- PDF readers:
  - Xodo
  - Foxit
   
Standard Notes is never among the apps I can share the text with.

**Comment by tasmo** [comment]
Unfortunate it is the same on iOS.

**Comment by 1MachineElf** [comment]
This problem has been around for years. Don't count on them ever prioritizing it.

https://github.com/standardnotes/mobile/issues/245

https://github.com/standardnotes/mobile/issues/11

**Comment by MatteoForrest** [comment]
It would be a great addition. And it's not against the SN "philosophy": it just helps the user take notes in a simpler and faster way (without a lot of copy-and-paste, clicking, and moving between apps).

**Comment by MrSuicideParrot** [comment]
Now that Standard Notes has introduced the Standard Notes clipper (https://standardnotes.com/help/usage/how-do-i-clip-web-pages) and since it is limited to desktop browsers, I think it is a great time to implement this feature.
 I know that this has being requested for 6 years. But we cannot use the clipper in mobile devices and  the share feature would be equivalent to the web clipper but for mobile.
